ext/nnstreamer/tensor_filter/tensor_filter_llama2.cc

```diff
@@ -166,17 +167,15 @@ TensorFilterLlama2c::invoke (const GstTensorMemory *input, GstTensorMemory *outp
 
   /** @todo Make out_info flexible */
   output[0].size = tokenizer->max_token_length * steps;
-  output_data = (char *) malloc (output[0].size);
-  len = strlen (result);
-  for (i = 0; i < len; i++) {
-    output_data[string_idx++] = result[i];
-  }
-  for (i = string_idx; i < (int) output[0].size; i++) {
-    output_data[i] = '\0';
+  output[0].data = allocate (output[0].size);
+
+  if (result) {
```

Comment:
Q. What does it mean if result is NULL?
In this case, is it right to allocate memory?
Don't need to throw exception?

Reply:
allocating empty string, to prevent pipeline freeze.